web/src/components/Navbar/index.js

```diff
@@ -1,88 +1,99 @@
 import React from 'react';
-import { Link } from 'react-router';
 import Scrollchor from 'react-scrollchor';
-import {
-  Collapse,
-  Navbar,
-  NavbarToggler,
-  NavbarBrand,
-  Nav,
-  NavItem,
-  NavLink,
-  UncontrolledDropdown,
-  DropdownToggle,
-  DropdownMenu,
-  DropdownItem } from 'reactstrap';
-import GlobalNavItem from '../global-nav-item/GlobalNavItem'
 
+const navbarItems = [
+  {
+    name: "Team",
+    scroll: "#team"
+  },
+  { 
+    name: "Partners",
+    scroll: "#partners"
 
-export default class ReactNavbar extends React.Component {
-  constructor(props) {
-    super(props);
+  },
+  { 
+    name: "Projects",
+    scroll: "#projects"
+  }
+];
+
+const navbarItemsList =
+navbarItems.map((data, index) => {
+  return (
+    <div className="nav__container">
+      <div className="nav__container-inner">
+        <a href={data.scroll} className="nav__list">{data.name}
```

Comment:
You can probably add scrollchor back once Ellie gets the front-end javascript working =)